src/PageLayout/PageLayout.tsx

```diff
@@ -197,6 +197,13 @@ const Header: React.FC<React.PropsWithChildren<PageLayoutHeaderProps>> = ({
   children,
   sx = {}
 }) => {
+  // Combine divider and dividerWhenNarrow for backwards compatibility
+  const dividerProp =
+    !isResponsiveValue(divider) && dividerWhenNarrow !== 'inherit'
```

Comment:
oof! full points for backwards compatibility but i don't think we should combine the two APIs

use the recommended API if given, fallback to deprecated one if not. either new or old, both

Reply:
Yup. As soon as you use a pass a "responsive object" value, you're opting into the new API and we ignore the `*whenNarrow` prop 👍